Failure in test-example-validation

[Unit and integration test breakages](https://travis-ci.org/ampproject/amphtml/jobs/243865808) on master seemingly due to #9997:

> Chrome 59.0.3071 (Linux 0.0.0) example autosuggest.amp.html should validate FAILED
	AssertionError: expected 'DISALLOWED_ATTR [src] amp-list\nEXTENSION_UNUSED amp-bind' to equal ''
	    at /home/travis/build/ampproject/amphtml/test/integration/test-example-validation.js:120:37 <- /tmp/57e74a45ba6487b46c25745bb825b639.browserify:209513:38

/to @Gregable

--- a/extensions/amp-list/0.1/test/validator-amp-list.html
+++ b/extensions/amp-list/0.1/test/validator-amp-list.html
@@ -32,6 +32,12 @@
             src="https://data.com/articles.json?ref=CANONICAL_URL">
     <div></div>
   </amp-list>
+  <!-- Valid amp-list with amp-bind attributes -->
+  <amp-list width=10 height=10
+            src="https://data.com/articles.json?ref=CANONICAL_URL"
+            [src]="foo.bar" [state]="baz.qux">
+    <div></div>
+  </amp-list>
   <!-- Invalid: width is mistyped. -->
   <amp-list src="https://data.com/articles.json?ref=CANONICAL_URL"
             wdith=10 height=10>

--- a/extensions/amp-list/0.1/test/validator-amp-list.out
+++ b/extensions/amp-list/0.1/test/validator-amp-list.out
@@ -1,5 +1,6 @@
 FAIL
 amp-list/0.1/test/validator-amp-list.html:31:2 The tag 'amp-list' requires including the 'amp-list' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
-amp-list/0.1/test/validator-amp-list.html:36:2 The attribute 'wdith' may not appear in tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
-amp-list/0.1/test/validator-amp-list.html:41:2 The mandatory attribute 'src' is missing in tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
-amp-list/0.1/test/validator-amp-list.html:46:2 The implied layout 'CONTAINER' is not supported by tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_LAYOUT_PROBLEM]
+amp-list/0.1/test/validator-amp-list.html:36:2 The tag 'amp-list' requires including the 'amp-list' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
+amp-list/0.1/test/validator-amp-list.html:42:2 The attribute 'wdith' may not appear in tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
+amp-list/0.1/test/validator-amp-list.html:47:2 The mandatory attribute 'src' is missing in tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_TAG_PROBLEM]
+amp-list/0.1/test/validator-amp-list.html:52:2 The implied layout 'CONTAINER' is not supported by tag 'amp-list'. (see https://www.ampproject.org/docs/reference/components/amp-list) [AMP_LAYOUT_PROBLEM]

--- a/extensions/amp-list/validator-amp-list.protoascii
+++ b/extensions/amp-list/validator-amp-list.protoascii
@@ -44,6 +44,9 @@ tags: {  # <amp-list>
   # TODO(gregable): Implement validation that requires the template attr value
   # to reference the id of an existing template element.
   attrs: { name: "template" }
+  # <amp-bind>
+  attrs: { name: "[src]" }
+  attrs: { name: "[state]" }
   attr_lists: "extended-amp-global"
   amp_layout: {
     supported_layouts: FILL

